MkDocs/apm_8.8/docs/matrix.md

```diff
@@ -0,0 +1,100 @@
+# Asset Matrix
+
+Maximo HP Utilities includes matrix view for determing the distribution of assets based on scores and then take action to avoid imminent failures.
+
+In this exercise the Reliability Engineer uses HP Utilities to:
+	
+1. [View Matrix for Criticality and End of life, Criticality and Risk, Criticality and Health](#view-matrix-for-criticality-and-end-of-life-criticality-and-risk-criticality-and-health)
+2. [View the drilled in assets list and take action](#view-the-drilled-in-assets-list-for-one-distribution)
+3. [Config Matrix version](#config-matrix-version)
+
+
+## Pre-requisites 
+
+This lab requires the following
+
+- A working Maximo Application Suite (MAS) environment with Maximo Manage, or another Enterprise Asset Management (EAM) system, Maximo Health, Maximo Health and Predict – Utilities (HPU), and Maximo Optimizer installed
+- Make sure the exercise [Understand Health and Predict - Utilities Models](hpu_models.md) is done, so that Sample ST(Substation Transformer) assets have the EOL/Risk/Health scores
+- Members of the EUSCORING security group can configure settings for the matrix(exercise 3), such as which version of the matrix is the default version and the colors and placement of cells.
+
+Please note that the MAS Worldwide (WW) demo environment is NOT a suitable environment for this lab.  The WW demo environment is shared, and making ANY changes to that environment will impact other users’ ability to demonstrate MAS.
+
+## Introduction
+
+This lab is intended to demonstrate the Matrix view capability in HPU which is another view besides the existing Grid view, Map view, Charts View. It allows Reliability engineer to esaily idenfity the high risk assets from a matrix distribution view and take actions based on the distributions.
+Reliability engineer can also easily config the different matrix versions, like to change the default version, disable version, change the matrix cells' color(category) etc.
+
+
+
+##  Navigate to Health and Predict - Utilities
+
+`Action:`
+
+1. Navigate to Health and Predict - Utilities (HPU). From the main MAS page, select the `Industry solutions` tab.
+![setup assets](/img/apm_8.7/AIO 1.png)
+
+2. Then click `Health and Predict – Utilities` menu.
+![setup assets](/img/apm_8.7/AIO 2.png)	 
+
+
+
+## View Matrix for Criticality and End of life, Criticality and Risk, Criticality and Health
+
+`Action:`
+1. From the main grid view, click the Matrix tab, you will see the matrix, each cell contains the assets count which fall in the x-axis and y-axis range.
+   For example, the red cell in the top right corner shows the assets that End of Life is high and Criticality is also High which means high risks.
```

Comment:
Shouldn't we add a step here to tell them to select an x/y configuration for the matrix like `health and criticality`

Reply:
x/y configuration is not supported in 8.8, it should be released in v8.9